nav-challenge/src/components/pages/signInPage/SignInPage.tsx

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
Do we need this files?

Reply:
this is just a dummy, so i don't get lint errors until component is finished